Main build KO due to Out of date version of GraalVM detected: GraalVM 22.1.0 Java 17

```
Error: #21 81.92 [ERROR]     [error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: java.lang.IllegalStateException: Out of date version of GraalVM detected: GraalVM 22.1.0 Java 17 CE (Java Version 17.0.3+7-jvmci-22.1-b06). Quarkus currently supports 22.3.0. Please upgrade GraalVM to this version.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,7 +8,7 @@ RUN yarn install --mode=skip-build
 RUN KAOTO_API="" yarn run build
 
 #Backend build
-FROM quay.io/quarkus/ubi-quarkus-native-image:22.1-java17 as BACKEND
+FROM quay.io/quarkus/ubi-quarkus-native-image:22.3-java17 as BACKEND
 ARG API_TAG
 USER root
 RUN microdnf install git curl

